pkg/daemon/daemon.go

```diff
@@ -203,6 +205,16 @@ const (
 	// Where nmstate writes the link files if it persisted ifnames.
 	// https://github.com/nmstate/nmstate/blob/03c7b03bd4c9b0067d3811dbbf72635201519356/rust/src/cli/persist_nic.rs#L32-L36
 	systemdNetworkDir = "etc/systemd/network"
+
+	// prefetchTimeout is the timeout for the prefetch process.
+	prefetchTimeout = 10 * time.Minute
+
+	// minFreeStorageAfterPrefetch is the minimum amount of storage in bytes available on the root filesystem
+	// after prefetching images.
+	minFreeStorageAfterPrefetch int64 = 32 * 1024 * 1024 * 1024 // 32GB
+
+	// authFilePath is the path to the auth file used by the prefetch process.
+	authFilePath = "/var/lib/kubelet/config.json"
```

Comment:
This seems to be used at multiple places now. We can also move it to pkg/daemon/constants/constants.go and use the authFilePath  var at other places as well https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/update.go#L420, https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/update.go#L2225